app/controllers/api/functions.php

```diff
@@ -786,22 +772,6 @@
             }
         }
 
-        /**
-         * TODO Should we update also the function collection with the scheduleUpdatedAt attr?
-         */
-
-        $schedule = $dbForConsole->getDocument('schedules', $function->getAttribute('scheduleId'));
-
-        $active = !empty($function->getAttribute('schedule'));
-
-        if ($active) {
-            $schedule->setAttribute('resourceUpdatedAt', datetime::now());
-        }
-
-        $schedule->setAttribute('active', $active);
-
-        Authorization::skip(fn () => $dbForConsole->updateDocument('schedules', $schedule->getId(), $schedule));
```

Comment:
why did we remove this ? How is the `active` attribute set when there is a new deployment ?

Reply:
Okay so is the builds worker updating the schedule ?

Reply:
Correct: https://github.com/appwrite/appwrite/blob/567cb01b9940948575abb08a1654b92cc61fe6d9/app/workers/builds.php#L214

Reply:
okay cool 👍